Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7
```

Comment:
We may need to install "our" mercurial (with robustcheckout) to make the worker more robust.

Reply:
do you have an example where we already do this?

Reply:
Looks like we already have `robustcheckout.py` vendored in in this repo. This may be unnecessary. I'll let @escapewindow or @Callek weigh on this.

Reply:
We do have robustcheckout, but the puppet configs do install `py27_mercurial`. We likely need to do the same. (Mercurial 5.0+ has [beta py3 support](https://www.mercurial-scm.org/wiki/Python3#Status), but we haven't verified that it works, and we haven't decided we want to switch to using beta quality mercurial for treescript. We could put in time testing this, or we can install the mercurial deb.)

Reply:
lets not try to do too much and do small steps. i will install mercurial via apt-get (as it is done in puppet) and then we can look in separate issue into 5.0+/py3 support.

@escapewindow sounds ok?

Reply:
Perfect.